src/conn/conn_cache.c

```diff
@@ -261,23 +41,16 @@ __wti_cache_config(WT_SESSION_IMPL *session, const char *cfg[], bool reconfig)
         conn->cache_size = 0;
 
     /*
-     * Always setup the local cache - it's used even if we are participating in a shared cache.
+     * If not using a shared cache configure the cache size, otherwise check for a reserved size.
+     * All other settings are independent of whether we are using a shared cache or not.
      */
-    WT_RET(__cache_config_local(session, now_shared, cfg));
-    if (now_shared) {
-        WT_RET(__wti_cache_pool_config(session, cfg));
-        WT_ASSERT(session, F_ISSET(conn, WT_CONN_CACHE_POOL));
-        if (!was_shared)
-            WT_RET(__wti_conn_cache_pool_open(session));
+    if (!now_shared) {
+        WT_RET(__wt_config_gets(session, cfg, "cache_size", &cval));
+        conn->cache_size = (uint64_t)cval.val;
     }
-
-    /*
-     * Resize the thread group if reconfiguring, otherwise the thread group will be initialized as
-     * part of creating the cache.
-     */
-    if (reconfig)
-        WT_RET(__wt_thread_group_resize(session, &conn->evict_threads, conn->evict_threads_min,
-          conn->evict_threads_max, WT_THREAD_CAN_WAIT | WT_THREAD_PANIC_FAIL));
+    /* Set config values as percentages. */
+    WT_RET(__wt_config_gets(session, cfg, "cache_overhead", &cval));
+    cache->overhead_pct = (u_int)cval.val;
```

Comment:
No action required, I'm just thinking out loud here; 
We discovered during this PR that `cache_size` and `overhead_pct` **must** be set before we call functions like `__evict_config_abs_to_pct` during eviction configuration, but this is far from obvious. Is there something we can do to make this dependency more obvious in future? Maybe it's enough to know cache must be configured before evict is configured

Reply:
I think it should be enough that cache should be configured before evict.